qiime2/sdk/plugin_manager.py

```diff
@@ -380,6 +380,12 @@ def get_directory_format(self, semantic_type):
             if semantic_type <= artifact_class_record.semantic_type:
                 return artifact_class_record.format
 
+        # TODO: We need a good way to tell if a semantic type is registered but
+        # does not have a directory format. The previous error was causing a
+        # lot of confusion, but this one is also not entirely accurate.
```

Comment:
```suggestion
        # lot of confusion.
```

I think it is accurate now, right?